plugins/win-capture/graphics-hook/CMakeLists.txt

```diff
@@ -15,12 +15,13 @@ add_library(_graphics-hook INTERFACE)
 
 target_sources(
   _graphics-hook
-  INTERFACE d3d8-capture.cpp
-            d3d9-capture.cpp
-            d3d9-patches.hpp
+  INTERFACE # cmake-format: sortable
             d3d10-capture.cpp
             d3d11-capture.cpp
             d3d12-capture.cpp
+            d3d8-capture.cpp
+            d3d9-capture.cpp
+            d3d9-patches.hpp
```

Comment:
This sorting is unfortunate.

Reply:
Indeed, but it is correct given a character-based sorting algorithm.

Reply:
I suppose we either sort it manually, or just take this change as-is.

Reply:
We kinda accepted the "unnatural" sorting introduced by `cmake-format` in other places, so accepting it here would just be consistent. 😄

Reply:
It's fine.